Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp

```diff
@@ -128,7 +128,6 @@ void Connection::platformInitialize(Identifier&& identifier)
 void Connection::platformInvalidate()
 {
 #if USE(GLIB)
-    // In the GLib platform the socket descriptor is owned by GSocket.
     m_socket = nullptr;
 #else
     if (m_socketDescriptor != -1)
```

Comment:
This line is now invalid:
```
C:/BW/PlayStation-Release-Build/build/Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp:133:28: error: invalid operands to binary expression ('UnixFileDescriptor' and 'int')
  133 |     if (m_socketDescriptor != -1)
      |         ~~~~~~~~~~~~~~~~~~ ^  ~~
```

Reply:
Sorry, I will land a follow-up

Reply:
https://github.com/WebKit/WebKit/pull/38142